inst/tinytest/test-nanotime.R

```diff
@@ -14,6 +15,10 @@ v <- as.nanotime(10^seq(0,6))
 expect_equal(plusOneVector(v), v+1)
 expect_equal(plusOneVector(v[1]), v[1]+1)
 
+v0 <- as.nanotime(as.integer64())               # length-0 vector
```

Comment:
... you could wrap this part in `if (requireNamespace(...))` and prefix the `as.integer64`.